src/tests/unit/tests/content/index.test.tsx

```diff
@@ -21,14 +21,14 @@ describe('content', () => {
         it(`can render ${path}`, () => {
             const ThisPage = contentPages.getPage(path);
             expect(ThisPage.displayName).toEqual('ContentPageComponent');
-            const result = shallow(<ThisPage deps={deps} />);
+            const renderResult = render(<ThisPage deps={deps} />);
             const headerExists =
-                result.find('h1').exists() ||
-                result.find('h2').exists() ||
-                result.find('Title').exists() ||
-                result.find('GuidanceTitle').exists();
+                renderResult.queryAllByRole('heading', { level: 1 }) ||
+                renderResult.queryAllByRole('heading', { level: 2 }) ||
+                renderResult.queryAllByRole('Title') ||
+                renderResult.queryAllByRole('GuidanceTitle');
```

Comment:
I ran this without these lines and it still passed so we can safely exclude them.
```suggestion
                renderResult.queryAllByRole('heading', { level: 1 }) ||
                renderResult.queryAllByRole('heading', { level: 2 });
```

Reply:
Excluded mentioned lines.